uat/testing-features/src/main/resources/greengrass/features/ggmq-1.feature

```diff
@@ -0,0 +1,32 @@
+@GGAD
+Feature: GGMQ-1
+
+  As a developer, I can configure my client agents to connect and use MQTT
+
+  Background:
+    Given my device is registered as a Thing
+    And my device is running Greengrass
+    And my MQTT Client Control is running
```

Comment:
what is this step going to do exactly? This isn't clear. This isn't something that an end-customer would understand. The point of using cucumber is that these steps are directly understandable by non-techincal readers.

Reply:
This isn't a step that a user would do, so let's hide it. In any step that needs the control to be running, verify that it is running, or start it if it isn't running. Then this step doesn't need to exist.

Reply:
Fixed c6275de8e27218f059bdf629c98e144c6891a718